client/src/components/categorical/category/index.tsx

```diff
@@ -114,8 +118,7 @@ class Category extends React.PureComponent<CategoryProps> {
     );
   }
 
-  // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types --- FIXME: disabled temporarily on migrate to TS.
-  handleColorChange = () => {
+  handleColorChange = (isColorAccessor: boolean) => {
```

Comment:
we need `isColorAccessor` in order to know whether the user is toggling on or off the color mode for analytics. We only want to trigger the event when the user is turning ON the color mode